README.md

```diff
@@ -182,11 +181,11 @@ const allUsers = await prisma.user.findMany({
 const filteredPosts = await prisma.post.findMany({
   where: {
     OR: [
-      { title: { contains: 'prisma' } },
-      { content: { contains: 'prisma' } },
+      { title: { contains: "prisma" } },
```

Comment:
Prettier?

Reply:
It just happened automatically 😅 should I revert?